scripts/st2bootstrap-deb.sh

```diff
@@ -96,6 +96,32 @@ setup_args() {
   fi
 }
 
+check_st2_host_dependencies() {
+  # Check that the following TCP ports are available.
+  # Abort the installation early if the required ports are being used by an existing process.
+
+  # nginx (80, 443), mongodb (27017), rabbitmq (4369, 5672, 25672), and st2 (9100-9102).
```

Comment:
Just noticed, there is also `PostgreSQL` service, which is used by `st2mistral`.